components/openindiana/nvidia-460/pkg5.fmris

```diff
@@ -0,0 +1 @@
+driver/graphics/nvidia-460
```

Comment:
You don't need this. It is only needed if multiple packages (fmris) will be delivered.

Reply:
Removed file pkg5.fmris from PR.